scripts/perftest/runAll.sh

```diff
@@ -20,11 +20,67 @@
 #
 #-------------------------------------------------------------
 
+#if [ "$1" == "" -o "$2" == "" ]; then echo "Usage: $0 <hdfsDataDir> <MR | SPARK | ECHO>   e.g. $0 perftest SPARK" ; exit 1 ; fi
 
-# Micro Benchmarks:
+# Example usage:
+# ./runAll.sh temp MR
 
-./scripts/perftest/MatrixMult.sh
-./scripts/perftest/MatrixTranspose.sh
+# First argument is optional, but can be e.g. perftestTemp
+TEMPFOLDER=$1
+if [ "$TEMPFOLDER" == "" ]; then TEMPFOLDER=temp ; fi
 
-# Algorithms Benchmarks:
+# Second argument is optional, but can be MR | SPARK | ECHO
+COMMAND=$2
+if [ "$COMMAND" == "" ]; then COMMAND=MR ; fi
+
+# Set properties
+export LOG4JPROP='conf/log4j-off.properties'
+export SYSDS_QUIET=1
+#export SYSTEMDS_ROOT=$(pwd)
+#export PATH=$SYSTEMDS_ROOT/bin:$PATH
+
+
+# Initialize Intel MKL
```

Comment:
After thinking more about it, I think it would be best to allow people to configure the use of BLAS from outside, without trying to hard-code it here.

Reply:
Maybe just take the three lines and them as a comment to runAll

Reply:
You mean these three lines?
- . ~/intel/bin/compilervars.sh intel64
- . ~/intel/oneapi/setvars.sh intel64
- . /opt/intel/bin/compilervars.sh intel64

Would it maybe be better if I just mention these three lines in the README near the requirements?

Reply:
either way is fine, if there is just a single README people will find it too. Otherwise having it in runAll as comment is also ok.